CHANGELOG/CHANGELOG-1.21.md

```diff
@@ -381,7 +381,6 @@ filename | sha512 hash
 - Migrate log messages in pkg/scheduler/{scheduler.go,factory.go} to structured logging ([#97509](https://github.com/kubernetes/kubernetes/pull/97509), [@aldudko](https://github.com/aldudko)) [SIG Scheduling]
 - Migrate proxy/iptables/proxier.go logs to structured logging ([#97678](https://github.com/kubernetes/kubernetes/pull/97678), [@JornShen](https://github.com/JornShen)) [SIG Network]
 - Migrate some scheduler log messages to structured logging ([#97349](https://github.com/kubernetes/kubernetes/pull/97349), [@aldudko](https://github.com/aldudko)) [SIG Scheduling]
-- NONE ([#97167](https://github.com/kubernetes/kubernetes/pull/97167), [@geegeea](https://github.com/geegeea)) [SIG Node]
```

Comment:
hmm I think we should leave 1.21 untouched for now since this is pre-release. we also recently just merged a new change to improve this filtering and parsing so hopefully this goes away before the release date.

Reply:
OK. i will revert the modification of 1.21 and remove none in 1.9.